tests/h/streamer/filter_test.py

```diff
@@ -3,6 +3,13 @@
 from h.streamer.filter import FilterHandler
 
 
+class FakeAnnotation:
```

Comment:
This is a mini-fake of `h.models.Annotation`. For the purposes of this test we don't need to save the annotation at all. Would it be better to use `Mock` for this?

Reply:
Is there / should there be a factory for this?